modules/ROOT/pages/release-notes.adoc

```diff
@@ -29,38 +29,20 @@ xref:index.adoc[Read more]
 
 *Fixed in this Release*
 
-- {jira-url}/CBG-461[*CBG-461*] Cobalt - cache enhancements- 404 error while requesting doc from the second sgw node
-- {jira-url}/CBG-458[*CBG-458*] cobalt - error count stats does not get incremented
-- {jira-url}/CBG-455[*CBG-455*] Inconsistent delta sync behavior regarding deletion
-- {jira-url}/CBG-449[*CBG-449*] _removed entry not sent with delta sync
-- {jira-url}/CBG-434[*CBG-434*] Build 2.6.0-104: Existing SG WRITE test with 100% RYOW shows suboptimal throughout in 4 node cluster compared to single node
-- {jira-url}/CBG-428[*CBG-428*] Packaged SG config tries connecting to couchbase://localhost instead of walrus
-- {jira-url}/CBG-424[*CBG-424*] Create conflict throws 400 error
-- {jira-url}/CBG-406[*CBG-406*] sgw fails to start with ssl enabled
-- {jira-url}/CBG-373[*CBG-373*] SGW shows wrong version number
-- {jira-url}/CBG-370[*CBG-370*] Sync gateway fails to start
-- {jira-url}/CBG-364[*CBG-364*] Odd SG Installation errors
-- {jira-url}/CBG-363[*CBG-363*] Return HTTP 413 when server rejects document due to size
-- {jira-url}/CBG-362[*CBG-362*] Inconsistent handling between views and N1QL for tombstone removals
-- {jira-url}/CBG-357[*CBG-357*] GoCB Bucket Incr amt=0 logic not taking 'def' into account
-- {jira-url}/CBG-342[*CBG-342*] GoCB Bucket Incr amt=0 logic not handling errors correctly
-- {jira-url}/CBG-324[*CBG-324*] Blip messages double-tagging user for redaction
-- {jira-url}/CBG-321[*CBG-321*] Improve sg-replicate logging integration
-- {jira-url}/CBG-319[*CBG-319*] Exit on error when retrieving ClusterCompatibility
-- {jira-url}/CBG-309[*CBG-309*] Don't log expected REST errors to sg_error.log
-- {jira-url}/CBG-306[*CBG-306*] Avoid panics as an error handling mechanism
-- {jira-url}/CBG-288[*CBG-288*] Investigate TestN1qlQuery failures
-- {jira-url}/CBG-286[*CBG-286*] SUBDOC operation attempted when server doesn't support them
-- {jira-url}/CBG-284[*CBG-284*] Retry loops for bucket update ops have no backoff and no max retry count
-- {jira-url}/CBG-280[*CBG-280*] Syncgateway doesn't import all documents while importing 50M docs ( build 2.1.3-4)
-- {jira-url}/CBG-279[*CBG-279*] go-blip leaks nextFrameToSend goroutine under aborted multi-frame send
-- {jira-url}/CBG-265[*CBG-265*] Post Upgrade fails to remove 2.0 era design docs
-- {jira-url}/CBG-264[*CBG-264*] Post Upgrade "fails" in a cluster with no query nodes
-- {jira-url}/CBG-253[*CBG-253*] Unused bulk_docs reserved sequences may not be released
-- {jira-url}/CBG-225[*CBG-225*] Unresponsive REST API on startup
-- {jira-url}/CBG-221[*CBG-221*] Admin/User views cannot handle both xattr and non-xattr simultaneously
-- {jira-url}/CBG-189[*CBG-189*] Compact() shouldn't trigger import errors
-- {jira-url}/CBG-159[*CBG-159*] CouchbaseServerVersion should use clusterCompatibility version
+- {jira-url}/CBL-415[*CBL-415*] strod produces locale-dependent results for encoding Fleece
```

Comment:
Looks like the script was run on the wrong project. `CBL` -> `CBG`

Reply:
Oops, yes

Reply:
updated.